packages/editor/src/client/routes/articleList.tsx

```diff
@@ -133,7 +133,7 @@ export function ArticleList() {
           data={articles}
           sortColumn={sortField}
           sortType={sortOrder}
-          rowClassName={rowData => (rowData?.id === highlightedRowId ? 'highlighted-row' : '')}
+          rowClassName={(rowData: ArticleRefFragment) => (rowData?.pending ? 'pending-approval' : '' || rowData?.id === highlightedRowId ? 'highlighted-row' : '')}
```

Comment:
@HanaAyman what if the row is pending approval and we want to highlight it? We need to be able to give the rows a basic color and still highlight them